CHANGELOG.md

```diff
@@ -25,6 +25,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.28.6] - 2024-06-26
+
+### Fixed
+- Updating the rounding to 4 decimal places instead of 2 and then not rounding if the 
+  value is above 99 to address the bug
```

Comment:
This changelog entry is not clear enough because it does not specify the command affected, so it will not be very helpful to somebody just reading the changelog. Here's what I would suggest:

```suggestion
### Fixed
- Improve info messages logged by `sat bootsys` in the `bos-operations` stage to
  use more precision when displaying percent successful values in order to avoid
  prematurely reporting that a BOS session is 100% complete when the actual
  percentage is between 99 and 100%.
```